docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: global
+title: Running Alluxio on Docker
+nickname: Alluxio on Docker
+group: Deploying Alluxio
+priority: 3
+---
+
+Alluxio can be run in a Docker container. This guide demonstrates how to run Alluxio
+in Docker using the Dockerfile that comes in the Alluxio github repository.
+
+# Prerequisites
+
+A Linux box. For the purposes of this guide, we will use a fresh EC2 machine running
+Amazon Linux. The machine size doesn't need to be large; we will use t2.small.
+
+# Launch a standalone cluster
+
+## Install Docker and Git
+
+```bash
+sudo yum install -y git docker
+sudo service docker start
+sudo usermod -a -G docker ec2-user
+```
+
+Finally, log out and log back in again to pick up the group changes
+
+## Clone the Alluxio repo
+
+```bash
+git clone https://github.com/Alluxio/alluxio.git
+```
+
+## Build the Alluxio Docker image
+
+```bash
+cd alluxio/integration/docker
+docker build -t alluxio .
+```
+
+By default, this will build an image for the latest released version of Alluxio. To build
+from a local Alluxio tarball, use `--build-arg`
+```bash
+docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+```
+
+## Run the Alluxio master
+
+```bash
+docker run -d alluxio master
+```
+
+## Run the Alluxio worker
+
+We need to tell the worker where to find the master. Set HOSTNAME to the hostname of
```

Comment:
Does this mean `ALLUXIO_MASTER_HOSTNAME` should be the hostname? Or, we should set `HOSTNAME` and then set `ALLUXIO_MASTER_HOSTNAME` on the docker command line?

Also, the worker container can be on a separate machine, right?